City-of-Pori/pori-kada
# PORI-99&100: Visitpori subtheme, header and footer theming.

Visitpori subtheme, header and footer theming.

To test:
1. drush cc all
2. go to appearance and enable & set default the Visitpori theme
3. go to frontpage and confirm that header and footer are following the layouts.
4. disable third level menu items from main navigation to check that the dropdowns look good.

When you resize your browser and after that mobile menu looks weird, hit refresh. The js is sometimes a bit slow to respond to resizing.
